static/js/avatar.js

```diff
@@ -90,7 +90,7 @@ export function build_user_avatar_widget(upload_function) {
     if (settings_account.user_can_change_avatar()) {
         return upload_widget.build_direct_upload_widget(
             get_file_input,
-            $("#user-avatar-upload-widget .image_file_input_error").expectOne(),
+            $("#avatar_image_file_input_error").expectOne(),
             $("#user-avatar-upload-widget .image_upload_button").expectOne(),
```

Comment:
So, the reason we had this selector structure is that we're 90% of the way through a refactoring that would allow us to only pass `#user-avatar-upload-widget` to `upload_widget.build_direct_upload_widget`.  See https://github.com/zulip/zulip/pull/16194 -- @clarammdantas @Jagansivam28 FYI in case either of you have time to rebase/clean up that effort, or help review this PR.

@ArunSankarKs feel free to look at picking up that effort as part of this broader effort.